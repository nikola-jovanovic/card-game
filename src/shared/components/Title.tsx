import styled from 'styled-components'

export default styled.h1`
  color: #fff;
  margin: 0;
  margin-bottom: 10px;
`
