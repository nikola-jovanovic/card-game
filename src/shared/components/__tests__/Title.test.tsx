import { render, screen } from '@testing-library/react'
import React from 'react'

import Title from '../Title'

describe('Title', () => {
  it('render', () => {
    const { container } = render(<Title>title</Title>)

    expect(screen.getByRole('heading', { name: /title/i, level: 1 })).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })

  it('render as h2', () => {
    render(<Title as='h2'>title</Title>)

    expect(screen.getByRole('heading', { name: /title/i, level: 2 })).toBeInTheDocument()
  })
})