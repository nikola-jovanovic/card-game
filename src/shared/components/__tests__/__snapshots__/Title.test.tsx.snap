// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title render 1`] = `
.c0 {
  color: #fff;
  margin: 0;
  margin-bottom: 10px;
}

<div>
  <h1
    class="c0"
  >
    title
  </h1>
</div>
`;
